sql/sql_connect.cc

```diff
@@ -947,7 +947,7 @@ static bool login_connection(THD *thd, bool extra_port_connection) {
     thd->reset_stats();
 
     // Updates global user connection stats.
-    increment_connection_count(thd, true);
+    increment_connection_count(*thd, true);
```

Comment:
Wow, so how did it even compile before? I see no overloads, nothing

Reply:
It compiles without a single warning and calls
```
  THD::THD(bool enable_plugins = true);
```
This is the second similar issue in this code. At https://github.com/percona/percona-server/commit/13f1823397556b3f5014aca75b5630e5569a4537 I fixed:
```
-      update_global_thread_stats_with_thread(thd, &thread_it->second, now);
+      update_global_thread_stats_with_thread(*thd, &thread_it->second, now);
```

Reply:
Can we mark that constructor explicit?

Reply:
Done and one more issue was found.